jenkins-builds/6.0.UP04-17.08-gpu

```diff
@@ -46,6 +46,7 @@
  PyExtensions-3.5-CrayGNU-17.08.eb
  QuantumESPRESSO-6.1.0-CrayIntel-17.08.eb           --set-default-module
  R-3.4.2-CrayGNU-17.08.eb                           --set-default-module
+ RDock/RDock-2013.1-CrayGNU-17.08.eb                --set-default-module
```

Comment:
maybe you should remove`RDock/` here? and also for `-mc`?

Reply:
👍